### ISSUE TITLE
Command line arguments missing

### ISSUE THREAD

**Issue by jhg03a** [derekparker/delve#629]
1. What version of Delve are you using (`dlv version`)?
   Delve Debugger
   Version: 0.11.0-alpha
   Build: af4798e2a91f2e57aa16fb38ba0275936cbdb2a1
2. What version of Go are you using? (`go version`)?
   go version go1.6.1 windows/amd64
3. What operating system and processor architecture are you using?
   Windows 10 Pro x64
4. What did you do?

```
package main

import (
    "fmt"
    "os"
)

func main() {
    fmt.Println(os.Args)
}
```

```
dlv debug -- foo bar
(dlv) c
[C:\repos\...snip...\go\scratch\debug]
Process 7432 has exited with status 0
```

```
PS C:\repos\...snip...\go\scratch> .\scratch.exe foo bar
[C:\repos\...snip...\go\scratch\scratch.exe foo bar]
```
1. What did you expect to see?
   [C:\repos...snip...\go\scratch\debug foo bar]
2. What did you see instead?
   [C:\repos...snip...\go\scratch\debug]

**Comment by aarzilli** [comment]
Is this a PowerShell thing or does it happen with cmd.exe also?

**Comment by jhg03a** [comment]
It happens with both CMD and PS.

**Comment by aarzilli** [comment]
Thanks.

**Comment by aarzilli** [comment]
I can't reproduce this on master, can you update, rebuild and try again?

**Comment by jhg03a** [comment]
It's reproducible for me with the head of master.

**Comment by jhg03a** [comment]
I found this as a byproduct of trying to get things to work with go-debug, but can't get it to work right on the CLI.

**Comment by aarzilli** [comment]
The commit that should fix this is 60946a759c03e68520b4e223cea9f79259612a21, the build you specified in the first message does not have that commit.

**Comment by aarzilli** [comment]
This bug can't be reproduced and it's been inactive for a while, closing.
